src/task/TaskRunner.ts

```diff
@@ -63,6 +63,10 @@ export class TaskRunner {
     );
   }
 
+  get isPolling() {
+    return this.poller.isPolling;
```

Comment:
Shouldn't this be `this.poller.isPolling()` or this is a TS convenience?

Reply:
getters are not accessed as methods in js